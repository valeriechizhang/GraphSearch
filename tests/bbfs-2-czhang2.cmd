--energy 20000 --end-x=7 --end-y=0