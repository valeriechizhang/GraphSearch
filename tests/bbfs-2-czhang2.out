Solution steps: ['E', 'E', 'E', 'E', 'E', 'E', 'E']
Solution cost: 15207
Number of states considered: 16

Frontier:
Pos=(4, 0) Moves=['E', 'E', 'E', 'E'] Cost=9905
Pos=(3, 1) Moves=['N', 'E', 'E', 'E'] Cost=4
Pos=(2, 2) Moves=['N', 'N', 'E', 'E'] Cost=4
Pos=(1, 3) Moves=['N', 'N', 'N', 'E'] Cost=4
Pos=(0, 4) Moves=['N', 'N', 'N', 'N'] Cost=4
Pos=(4, 0) Moves=['E', 'E', 'E'] Cost=5302
Pos=(5, 1) Moves=['E', 'E', 'S'] Cost=3
Pos=(6, 2) Moves=['E', 'S', 'S'] Cost=3
Pos=(7, 3) Moves=['S', 'S', 'S'] Cost=3

Closed List:
Pos=(0, 0) Moves=[] Cost=0
Pos=(0, 1) Moves=['N'] Cost=1
Pos=(1, 0) Moves=['E'] Cost=2402
Pos=(0, 2) Moves=['N', 'N'] Cost=2
Pos=(1, 1) Moves=['N', 'E'] Cost=2
Pos=(2, 0) Moves=['E', 'E'] Cost=4903
Pos=(0, 3) Moves=['N', 'N', 'N'] Cost=3
Pos=(1, 2) Moves=['N', 'N', 'E'] Cost=3
Pos=(2, 1) Moves=['N', 'E', 'E'] Cost=3
Pos=(3, 0) Moves=['E', 'E', 'E'] Cost=7404
Pos=(7, 0) Moves=[] Cost=0
Pos=(7, 1) Moves=['S'] Cost=1
Pos=(6, 0) Moves=['E'] Cost=300
Pos=(7, 2) Moves=['S', 'S'] Cost=2
Pos=(6, 1) Moves=['E', 'S'] Cost=2
Pos=(5, 0) Moves=['E', 'E'] Cost=2801
